tests/pytests/unit/modules/test_at.py

```diff
@@ -0,0 +1,236 @@
+"""
+    :codeauthor: Rupesh Tare <rupesht@saltstack.com>
+
+    TestCase for the salt.modules.at module
+"""
+
+
+import pytest
+
+import salt.modules.at as at
+import salt.utils.path
+from tests.support.mock import MagicMock, patch
+
+
+@pytest.fixture
+def configure_loader_modules():
+    return {at: {}}
+
+
+atq_output = {
```

Comment:
We want to avoid using mutable objects at the state/global level. Can you please use a pytest fixutre instead. This is documented here: https://docs.saltproject.io/en/master/topics/development/tests/index.html#module-global-level-variables

Reply:
Thanks, I'll give it another look.